konfera/models/ticket_type.py

```diff
@@ -24,6 +25,13 @@ class TicketType(FromToModel):
     def __str__(self):
         return self.title
 
+    def status(self):
+        if timezone.now() < self.date_from:
+            return _('Not available yet')
+        elif self.date_to < timezone.now():
+            return _('Expired')
+        return _('Active')
```

Comment:
Please add constants. In test you can import constant and compare against it in tests as well, this will work no matter of the language...

```
NOT_AVAILABLE = 'na'
ACTIVE = 'active'
...
```

We might have a dict for resolution:

```
STATUS = {
    NOT_AVAILABLE:  _('Not available yet')
    ACTIVE: _('Active')
    ....
}
```

and function (get_status or something similar) that return translation string, that will be used in admin, or anywhere else where needed...